Tutorial HTML/Html 01/README.md

```diff
@@ -0,0 +1,85 @@
+Pentru a continua ne este indeajuns sa intelegem pe scurt rolul fiecarui limbaj intr-o aplicatie
+Nu ne vom arunca prea mult inca in detalii la acest punct.
+
+- *Am adaugat un link catre wikipedia cu niste explicatii, pentru cei care doresc insa acest lucru.*
+
+#### - HTML ( Hyper text markup language )
+[HTML Wikipedia](https://ro.wikipedia.org/wiki/HyperText_Markup_Language)
+Este folosit pentru a crea pagini web simple ce pot fi afișate într-un browser.
+Scopul HTML este  in general prezentarea informațiilor – paragrafe fonturi, tabele ș.a.m.d si este limitat la acest lucru de unul singur.
+
+#### - CSS ( Cascading Style Sheets )
+[CSS Wikipedia](https://ro.wikipedia.org/wiki/Cascading_Style_Sheets)
+ CSS formateaza HTML-ul scris, si cu ajutorul unui set de prorietati se pot pozitiona elemente in pagina, se pot face animatii, ș.a.m.d
+
+#### - JS (JavaScript)
+ ##### A nu se confunda cu Java ("Java is to JavaScript what car is to carpet")
+
+[JS Wikipedia](https://ro.wikipedia.org/wiki/JavaScript)
+ Este un limbaj de programare ce poate fi folosit pentru realizarea logicii unei pagini web ( formulare autentificare, ce se intampla cand apesi un buton, ș.a.m.d) precum si diverse aplicatii, jocuri, animatii  ș.a.m.d.
+
+Aceste 3 lucruri combinate pot avea ca rezultat o [aplicatie web](https://ro.wikipedia.org/wiki/Aplica%C8%9Bie_web)
+
+## HTML
+
+O aplicatie web nu poate functiona fara macar o boaba de HTML asa ca vom incepe cu acest lucru.
+
+- Sa presupunem ca dorim sa creeam o pagina web care sa contina poezii
+Avem asignata cate o poezie pentru fiecare pagina, si deasemenea dorim sa afisam un tabel cu date despre fiecare poezie pe pagina ei
+Deasemenea vom dori ca aceasta pagina web sa contina o sectiune ce va permite navigarea intre poezii,
+
+HTML este un limbaj de interpretare.
+Acest lucru ne spune ca HTML e un format de text cu anumite componente ce sunt interpretate de browser pentru a fi afisate in mod corespunzator.
+
+De exemplu ne putem gandi aici la relatia dintre un cititor si un poet.
+Poetul scrie o poezie de sucess cu gandul la dragoste, si fericire, dintr-o experienta trecuta.
+Vine citirorul, se apuca de citit si datorita experientelor sale diferite acesta interpreteaza poezia ca fiind una trista, si o povesteste mai departe ca atare.
+In cazul de fata cititorul nostru este browserul, iar scriitorul fiind cel care creaza o pagina web
+
+Un fisier HTML contine comunica cu browserul prin taguri *(mai corect spus browserul ia aceste taguri si le interpreteaza ca atare)* 
+
+### Taguri HTML
+Un element HTML este format din un `tag de inceput` niste text si un `tag de final`
+Ele au rolul de a specifica browserului modul in care dorim ca textul pe care il dam sa fie interpretat
+De exeplu `<p> Sunt un paragraf ! </p>` va face browserul sa interpreteze acest text ca un paragraf
+
+- Un tag HTML este un text de forma `<` `final?` `numetag` `optiuni` `>`
```

Comment:
Explicatia text pare mult mai complicata decat e de fapt. Oare sa adaugam si o ilustratie de genul: 
![image](https://user-images.githubusercontent.com/11327761/86396331-6731ca80-bcaa-11ea-9893-9c067dc1f9e9.png), cred ca ar prinde mai usor

Reply:
Sa vad in ce le pot face

Reply:
Eu ti-am dat printscreen din [slides.com](https://slides.com). Au basic stuff, linii, cod, text, etc. Poti face pe un background alb, si sa faci printscreen-> save as Image.

O alta idee dar nu cred ca avem timp, ar fi sa facem noi desenele ca aici https://hacks.mozilla.org/2019/11/announcing-the-bytecode-alliance/ 😍